exercises/concept/weighing-machine/.meta/Exemplar.cs

```diff
@@ -1,80 +1,40 @@
 using System;
 
-enum Unit
-{
-    Pounds,
-    Kilograms
-}
-
 class WeighingMachine
 {
-    private const decimal POUNDS_PER_KILOGRAM = 2.20462m;
+    private int _weight;
 
-    private decimal inputWeight;
+    public WeighingMachine(int precision)
+    {
+        Precision = precision;
+    }
 
-    public Unit Unit { get; set; } = Unit.Kilograms;
+    // Goal: getter-only property initialized in constructor
```

Comment:
All the `// Goal: ...` comments in this file can be removed. I just added them to show what they were used for.

```suggestion
```